content/1.7/back-office/international/locations/add-new-state.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Add new state
+weight: 1
+---
+
+# Add new state
+
+While adding or editing a state, 5 fields below are shown:
+
+ - **Name** - this is a mandatory field. A tooltip for this field shows: "Provide the state name to be displayed in addresses and on invoices.". When trying to save with the field empty will show an error message: "The name field is required.".
```

Comment:
```suggestion
 - **Name** - this is a mandatory field. A tooltip for this field is displayed: "Provide the state name to be displayed in addresses and on invoices.". When trying to save and if the field is empty, the following error message is displayed: "The name field is required.".
```

Reply:
what happens if I try to save without filling this field?

Reply:
Is there something more to be added? As I've added already the error message when field is empty